hypha/apply/funds/templates/funds/includes/admin_primary_actions.html

```diff
@@ -14,6 +14,12 @@ <h5>Actions to take</h5>
 
 <a data-fancybox data-src="#screen-application" class="button button--bottom-space button--primary button--full-width {% if screening_form.should_show %}is-not-disabled{% else %}is-disabled{% endif %}" href="#">Screen application</a>
 
+{% if object.is_determined %}
+    {% with determination=object.determinations.final.first %}
```

Comment:
we want the latest determination here i think so `last`?

Reply:
I don't think it would make much difference to use first or last as the view guarentees there's only one determination but I've changed this to be consistent with generic_primary_actions.html